src/components/SetupFundersPage/SetupFundersPage.story.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { SetupFundersPage } from "./index";
+
+storiesOf("Components|SetupFundersPage", module).add("Default", () => (
```

Comment:
This story doesn't work – I recommend removing stories on the graphql pages until we work out a way of mocking it.